source/clear-linux/get-started/virtual-machine-install/vmware-player.rst

```diff
@@ -1,116 +1,528 @@
 .. _vmware-player:
 
-Use VMware\* Player
-###################
+Run Clear Linux as a VMware\* Workstation Player guest OS
+#########################################################
 
-This section explains how to run Clear Linux OS for Intel® Architecture
-within a `VMware Player`_ environment.
+`VMware Workstation 14 Player`_ is a type 2 hypervisor which runs on top of another
+operating system such as Windows or Linux.  Creation, configuration, management, and execution of virtual
+machines are done on the local system.  For the cloud version of VMware, see 
+:ref:`vmware-esxi`.
 
-Please ensure you have enabled `Intel® Virtualization Technology
-<http://www.intel.com/content/www/us/en/virtualization/virtualization-technology/intel-virtualization-technology.html>`_
-(Intel® VT) and `Intel® Virtualization Technology for Directed I/O
-<https://software.intel.com/en-us/articles/intel-virtualization-technology-for-directed-io-vt-d-enhancing-intel-platforms-for-efficient-virtualization-of-io-devices>`_
-(Intel® VT-d) in your BIOS/UEFI firmware configuration.
+This section explains how to run |CLOSIA| as a virtual machine using the 
+VMware Workstation 14 Player hypervisor.
 
-Install VMware Player
-=====================
+There are 2 ways to create a |CL| :abbr:`VM (Virtual Machine)` to run in VMware:
```

Comment:
Incorrect verb conjugation:  There are two ways to create a |CL| :abbr:`VM (Virtual Machine)` running in VMware: